source/languages/en/riak/ops/advanced/riak-control.md

```diff
@@ -14,181 +14,151 @@ moved: {
 Riak Control is a web-based administrative console for inspecting and
 manipulating Riak clusters.
 
-{{#1.4.0-}}
-See the below video for a quick introduction to Riak Control and its features.
-
-<div style="display:none" class="iframe-video" id="http://player.vimeo.com/video/38345840"></div>
-{{/1.4.0-}}
-
 ## Requirements
 
 Though Riak Control [is maintained as a separate
-application](https://github.com/basho/riak_control), the necessary code for
-Control ships with versions of Riak 1.1 and above, and requires no additional
-installation steps.
-
-It is strongly recommended that SSL be enabled for Riak Control. SSL is
-required unless you explicitly set `{auth, none}`. SSL can be enabled in the
-[[configuration files]].
-
-## Setting up Riak Control
+application](https://github.com/basho/riak_control), the necessary code
+for it ships with versions of Riak 1.1 and above and requires no
+additional installation steps.
 
-### Enabling SSL and HTTPS
+<div class="note">
+<div class="title">Note on SSL</div>
+We strongly recommend that you enable SSL for Riak Control. It is
```

Comment:
Might be useful to say that if the ``userlist`` method of authentication is selected, and ``ssl`` is disabled, Riak Control won't work.  This has come up a few times on the mailing list.